src/Serilog.Sinks.Raygun/Sinks/Raygun/RaygunSink.cs

```diff
@@ -72,34 +101,62 @@ public class RaygunSink : ILogEventSink
             string groupKeyProperty = "GroupKey",
             string tagsProperty = "Tags",
             string userInfoProperty = null,
-            Action<OnBeforeSendArguments> onBeforeSend = null)
+            Action<OnBeforeSendArguments> onBeforeSend = null
+#if NET || NETSTANDARD
```

Comment:
Is it a good idea for the API to change based on the language runtime version?

Reply:
Because the classes only exist for asp.net core library, there isn't any other way, but we will look at addressing this in the future.